docs/tutorials/using_apis/mesh/thread_intro.md

```diff
@@ -55,9 +57,9 @@ The Mbed OS Mesh API allows you to define 3 types of devices:
 
 In addition the Nanostack API allows you to define the Full end device (FED) device type.
 
-In most cases, the REED, MED and SED configurations are enough to build a network. In a SED device, the radio is switched off during the sleep times. To take the full advantage of the SED configuration, the application developer must implement the sleep procedures in the device.
+In most cases, the REED, MED and SED configurations are enough to build a network.
 
-For an end device or a router example, see [mesh minimal example](https://github.com/ARMmbed/mbed-os-example-mesh-minimal).
+For an end device or a router example, see the [mesh minimal example](https://github.com/ARMmbed/mbed-os-example-mesh-minimal).
```

Comment:
We don't like to link to GitHub. Is there a Doxygen page or teams example to which we could link instead?

Reply:
In this case no there isn't.